src/transformers/onnx/features.py

```diff
@@ -142,17 +142,8 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=BartOnnxConfig,
         ),
-        "mbart": supported_features_mapping(
-            "default",
-            "default-with-past",
-            "causal-lm",
-            "causal-lm-with-past",
-            "seq2seq-lm",
-            "seq2seq-lm-with-past",
-            "sequence-classification",
-            "question-answering",
-            onnx_config_cls=MBartOnnxConfig,
-        ),
+        # BEiT cannot be used with the masked image modeling autoclass, so this feature is excluded here
+        "beit": supported_features_mapping("default", "image-classification", onnx_config_cls=BeitOnnxConfig),
```

Comment:
Since I had to edit the features here, I also went ahead and reordered all the models alphabetically since the list is now quite long and annoying to navigate

Reply:
This is hard to review as a diff, so I'll trust you didn't forget any of them ;-)

Reply:
Yeah sorry about that. I did a sanity check that all the features agree with those on the `main` branch:

```
from transformers.onnx import FeaturesManager

# From current branch
features_new = FeaturesManager._SUPPORTED_MODEL_TYPE
# From main brach
features_old = FeaturesManager._SUPPORTED_MODEL_TYPE

for k,v in features_new.items():
    # Skip beit since it's features are different on `main`
    if k == "beit":
        continue
    assert features_old[k].keys() == v.keys()
```